jquery.flexslider.js

```diff
@@ -1032,6 +1035,25 @@
       slider.vars.removed(slider);
     };
 
+	slider.destroy = function() {
+		var classNamespace = '.' + slider.vars.namespace; // Namespaced class selector
+		if (slider.vars.controlNav) slider.controlNav.closest(classNamespace + 'control-nav').remove(); // Remove control elements if present
+		if (slider.vars.directionNav) slider.directionNav.closest(classNamespace + 'direction-nav').remove(); // Remove direction-nav elements if present
+		if (slider.vars.pausePlay) slider.pausePlay.closest(classNamespace + 'pauseplay').remove(); // Remove pauseplay elements if present
+		slider.find('.clone').remove(); // Remove any flexslider clones
+		slider.unbind(slider.vars.eventNamespace); // Remove events on slider
+		if ( slider.vars.animation != "fade" ) slider.container.unwrap(); // Remove the .flex-viewport div
+		slider.container.removeAttr('style') // Remove generated CSS (could collide with 3rd parties)
+		slider.container.unbind(slider.vars.eventNamespace); // Remove events on slider
+		slider.slides.removeAttr('style'); // Remove generated CSS (could collide with 3rd parties)
```

Comment:
The removal of styles should come after the events are unbound and the timers stopped. Before a timer is unbound it could expire and start adding styles again. Just needs moving down a little in this function (applies to both sets of removeAttr('style') function calls).